src/GovUk.Education.ExploreEducationStatistics.Data.Services/TableBuilderService.cs

```diff
@@ -68,14 +73,15 @@ public TableBuilderService(
             _userService = userService;
             _releaseVersionRepository = releaseVersionRepository;
             _options = options.Value;
+            _locationOptions = locationOptions.Value;
         }
 
         public async Task<Either<ActionResult, TableBuilderResultViewModel>> Query(
             FullTableQuery query,
             CancellationToken cancellationToken = default)
         {
             return await FindLatestPublishedReleaseVersionId(query.SubjectId)
-                .OnSuccess(releaseVersionId => Query(releaseVersionId, query, null, cancellationToken));
+                .OnSuccess(releaseVersionId => Query(releaseVersionId, query, boundaryLevelId: null, cancellationToken));
         }
 
         public async Task<Either<ActionResult, TableBuilderResultViewModel>> Query(
```

Comment:
I'd be inclined to add a comment to the `boundaryLevelId` argument in this method - something like
``` long? boundaryLevelId, // remove in EES-XXXX```
in case that sticks around for longer than expected.